README.md

```diff
@@ -2,34 +2,55 @@
 
 # qubes-template-securedrop-workstation
 
-This work was inspired by and reuses code from the Whonix Qubes template: https://github.com/adrelanos/qubes-template-whonix
-It is a derivative work under the GPL license, version 3 (see the files `COPYING` and `GPLv3` for details)
-
+Repository for managing the TemplateVM RPM used by the [SecureDrop Workstation] in provisioning custom VMs.
 
 ## Build instructions
 
-Note that these instructions must be carried out on a Fedora-based Qubes VM. Building templates uses a substantial amount of disk space.
+Note that these instructions must be carried out on a Fedora-based Qubes VM.
+Building templates uses a substantial amount of disk space.
 
 ### Set up build VM:
-1. Create a fedora-31 based AppVM for building the templates
-2. Increase the disk size to at least 20GB (as the build uses over 10GB): qvm-volume extend sd-template-builder:private 15GB (if your VM is not named sd-template-builder, adjust that command)
-3. Import the QubesOS master key and the GPG key used to sign tags (see https://www.qubes-os.org/security/verifying-signatures/)
+Set up a long-lived VM that you can use for building SDW templates.
+You'll only need to perform this step once, although make sure to check
+whether your Fedora version remains current.
+
+1. Create an AppVM based on the most recent fedora release: `qvm-create --label purple --template fedora-33 sd-template-builder`
```

Comment:
recommend changing `fedora-33` to `fedora-XX` where XX is the latest version of fedora available (which you can see within the Qubes menu)